pkg/ansible/runner/runner.go

```diff
@@ -68,14 +68,18 @@ type cmdFuncType func(ident, inputDirPath string, maxArtifacts, verbosity int) *
 
```

Comment:
justing checking/fixing new issues introduced. 
Yep. I think it is valid :-)

Reply:
I disagree with the changes of having to assign to variables. What is the purpose of a change like this? I know it may make it easier to debug but it isn't technically superior. I don't like this because I would never write code like this.

Reply:
I agree with @jmrodri. If anything, this just obfuscates the issue from the linter and doesn't actually solve anything as far as I can tell.